client/mysqlbinlog.cc

```diff
@@ -2922,6 +2923,17 @@ static Exit_status dump_local_log_entries(PRINT_EVENT_INFO *print_event_info,
 
     Log_event* ev = Log_event::read_log_event(file, glob_description_event,
                                               opt_verify_binlog_checksum);
+
+    // If we're not in raw mode, then emit a warning in the event that we
```

Comment:
Just a small formatting remark, but the general style (at least in replication code) for formatting multi-line comments is to wrap it like:
```
/*
  line 1
  line 2
*/
```

Though note, there are a few "legacy" files in the replication code base which unfortunately deviate in their own way, and we try to just keep the formatting consistent per-file. `mysqlbinlog.cc` uses the above though.

Reply:
Thanks, will fix!